integration_tests/e2e/notifications-settings.cy.ts

```diff
@@ -99,7 +100,7 @@ context('A staff member can view their notification settings', () => {
     Page.verifyOnPage(NotificationManagePage)
     cy.task('verifySnooze').then((requests) => {
       expect(requests).to.have.length(1)
-      expect(requests[0].body).eq(`{"snoozeUntil":"${moment().add(12, 'days').format('YYYY-MM-DD')}"}`)
+      expect(requests[0].body).eq(`{"snoozeUntil":"${addDays(Date.now(), 12).toISOString().split('T')[0]}"}`)
     })
```

Comment:
shame the new way is so clunky

Reply:
I could have done a `format` here instead but would have still needed to then provide a `yyyy-MM-dd`, not sure if that was any better than doing the native `toISOString`